mm2src/coins/lp_coins.rs

```diff
@@ -268,6 +268,18 @@ pub type TxHistoryResult<T> = Result<T, MmError<TxHistoryError>>;
 pub type RawTransactionResult = Result<RawTransactionRes, MmError<RawTransactionError>>;
 pub type RawTransactionFut<'a> =
     Box<dyn Future<Item = RawTransactionRes, Error = MmError<RawTransactionError>> + Send + 'a>;
+// has takers pubkey
```

Comment:
In my opinion, these comments are not required. If you wish, it will be more informative to add corresponding comments to the `SendSwapPaymentArgs` structure fields like:
```rust
struct SendSwapPaymentArgs {
  /// This is either:
  /// * Taker's pubkey if this structure is used in [`SwapOps::send_maker_payment`].
  /// * Maker's pubkey if this structure is used in [`SwapOps::send_taker_payment`].
  other_pubkey: ...
}
```

What do you think?

Reply:
I think that's a great idea, actually! I don't like comments above the types either.
Will do the same thing with payment tx:
```rust
pub struct SendSpendPaymentArgs<'a> {
    /// This is either:
    /// * Taker's payment tx if this structure is used in [`SwapOps::send_maker_spends_taker_payment`].
    /// * Maker's payment tx if this structure is used in [`SwapOps::send_taker_spends_maker_payment`].
    pub other_payment_tx: &'a [u8],
    /// This is either:
    /// * Taker's pubkey if this structure is used in [`SwapOps::send_maker_spends_taker_payment`].
    /// * Maker's pubkey if this structure is used in [`SwapOps::send_taker_spends_maker_payment`].
    pub other_pubkey: &'a [u8],
}
```

Reply:
done